src/_includes/partials/author_bios/yeonjuan.md

```diff
@@ -0,0 +1 @@
+@eslint team alumnus. Passionate about JavaScript-based developer tools.
```

Comment:
Just to double-check, would you like to expand your bio here? This is pulled from GitHub but you can definitely add more information if you'd like.